src/pages/filter/ui/enum-panel.tsx

```diff
@@ -1,16 +1,18 @@
 import { ReactElement, useEffect, useState } from 'react'
+import { cloneDeep } from 'lodash'
```

Comment:
avoid importing from `lodash`, prefer to import from `lodash/%functionName%`.
The problem is, when you import something from lodash, you import the whole library, and it's enormously large. On the opposite, when you import a particular function from lodash like this - `import cloneDeep from 'lodash/cloneDeep'`, you import the only function you need. And this makes the bundle way lighter.